flink-table/flink-table-common/src/main/java/org/apache/flink/table/api/Schema.java

```diff
@@ -611,7 +611,9 @@ private void addResolvedColumns(List<Column> columns) {
                         } else if (c instanceof ComputedColumn) {
                             final ComputedColumn computedColumn = (ComputedColumn) c;
                             columnByExpression(
-                                    computedColumn.getName(), computedColumn.getExpression());
+                                    computedColumn.getName(),
+                                    new SqlCallExpression(
+                                            computedColumn.getExpression().asSerializableString()));
```

Comment:
Here, I modify the behavior by converting the resolvedExpression to an unresolvedExpression when using `Schema.newBuilder().fromResolvedSchema().build()`. This modification simplifies schema comparison during diff.